creusot/tests/should_fail/terminates/trait_def_and_impl_disagree.stderr

```diff
@@ -10,10 +10,10 @@ error: Expected `h` to be `#[pure]` as specified by the trait declaration
 22 |     fn h() {}
    |     ^^^^^^
 
-error: Expected `f` to be `#[terminates]` as specified by the trait declaration
-  --> trait_def_and_impl_disagree.rs:17:5
+error: Expected `g` to be `#[pure]` as specified by the trait declaration
+  --> trait_def_and_impl_disagree.rs:20:5
```

Comment:
Is this is an expected change in error?

Reply:
For some reason, iterators over syntax tree definitions changed the iteration order, which results in order changes in generated coma files and in printed errors.